knative-operator/deploy/resources/console_cli_download_kn.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: kn
+spec:
+  description: The OpenShift Serverless client `kn` is a CLI tool that allows you to fully manage OpenShift Serverless Serving and Eventing resources without writing a single line of YAML.
+  displayName: kn - OpenShift Serverless CLI
+  links:
+  - href: https://mirror.openshift.com/pub/openshift-v4/clients/serverless/0.11.0/kn-linux-amd64-0.11.0.tar.gz
+    text: Download kn for Linux
+  - href: https://mirror.openshift.com/pub/openshift-v4/clients/serverless/0.11.0/kn-macos-amd64-0.11.0.tar.gz
+    text: Download kn for macOS
+  - href: https://mirror.openshift.com/pub/openshift-v4/clients/serverless/0.11.0/kn-windows-amd64-0.11.0.zip
+    text: Download kn for Windows
```

Comment:
Added a commit which adds to-be-synced-at-mirror kn version for 1.4.0 release (as kn v0.11.0 tar ball at mirror will be synced after QA is done, which is past our code-freeze date)

Reply:
Another thing we could do here is keep the link of `latest` i.e. https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/ , which is sym-linked to latest available kn version at mirror.

Cc: @rhuss